bloom_nofos/nofos/api/schemas.py

```diff
@@ -0,0 +1,77 @@
+from ninja import ModelSchema, Schema
+from typing import List, Optional
+from nofos.models import Nofo, Section, Subsection
+
+
+class SubsectionSchema(ModelSchema):
+    class Config:
+        model = Subsection
+        model_fields = [
+            "name",
+            "html_id",
+            "order",
+            "tag",
+            "body",
+            "callout_box",
+            "html_class",
+        ]
+        model_fields_optional = ["html_class"]  # Fields that should have defaults
+
+
+class SectionSchema(ModelSchema):
+    subsections: List[SubsectionSchema]
+
+    class Config:
+        model = Section
+        model_fields = ["name", "html_id", "order", "has_section_page"]
```

Comment:
Can we do the same thing with creating a base schema for sections as well? Would be nice to see section metadata  before the list of subsections.